lib/TopBar/styles.scss

```diff
@@ -152,7 +159,7 @@ $top-bar-notification-height: 42px;
       color: white;
     }
     .top-bar__action,
-    .icon path {
+    .icon:not(.structure__save-btn) path {
```

Comment:
🔴  where is `.structure__save-btn` from? 

Seems like it's from something outside of GUI and shouldn't be added to the styles specifically.

Reply:
Yeah that's a good point. Will need to figure out a different way.